### DISCUSSION TITLE
Venus OS v3.50

### DISCUSSION THREAD

**Discussion by Ep1phany-CA** [mr-manuel/venus-os_dbus-serialbattery#101] [Q&A] [answered]
Venus OS v3.50 was just released with Gui V2. Will you (@mr-manuel) be be spending any time integrating the project into your gui-v2 project?

**Comment by mr-manuel** [comment]
It's in the [README.md](https://github.com/mr-manuel/venus-os_dbus-serialbattery?tab=readme-ov-file#guiv2) since almost a year...

**Comment by Ep1phany-CA** [comment]
I'm aware, and that wasn't my question. Your readme states "Many functions are not yet visible in the new GUI, also it is not sure that this is the way it will be implemented in future". So this was a general inquiry to see if you'll keep working on that integration or if you're done with the project.

**Reply by mr-manuel** [reply, marked as answer]
Currently I‘m working with Victron employees to find a solution for that, this takes and will take a lot of time. One plugin integration is no problem, but integrating more plugins together is. There is still no direction defined.

See https://github.com/victronenergy/gui-v2/issues/1071 and https://communityarchive.victronenergy.com/questions/245056/venus-os-modifying-gui-v2.html
